Add WA state and MA programs

MaxGhenis opened the pull request:
Fixes #810 by adding WA's capital gains tax and Working Families Tax Credit, and as a population impact drop-down item

Also adds more MA programs recently implemented in OpenFisca US

MaxGhenis commented on policyengine/countries/us/reform_parameters.yaml:
```diff
@@ -140,6 +140,26 @@ reforms:
+        label: Abolish WA Working Families Tax Credit
+        name: abolish_wa_wftc
+        unit: abolition
+        variable: wa_wftc_max_amount
```
Does this work, given [`wa_wftc_max_amount` is a scale parameter](https://github.com/PolicyEngine/openfisca-us/blob/master/openfisca_us/parameters/gov/states/wa/tax/income/credits/working_families_tax_credit/amount.yaml)?

MaxGhenis replied:
I should probably point it to a variable instead of a parameter then!

MaxGhenis commented:
The standard and itemized deduction submenus break for some reason. `aged_blind_std_age` works, but [`standard_deduction`](https://github.com/PolicyEngine/openfisca-us/blob/master/openfisca_us/parameters/gov/irs/deductions/standard/amount.yaml) and `aged_blind_std`, both of which are indexed on `filing_status`, break it. I haven't looked more into the itemized menu items. They appear to be correctly specified, so I'm not sure why it's breaking.

The WA and MA programs work though, I just want to dig a bit more into the results from the new policies before merging. Here are simulations of repealing them.

## WA Working Families Tax Credit

Expected top-line results.
<img width="1296" alt="image" src="https://user-images.githubusercontent.com/6076111/178293698-be5e26f5-cb85-48d3-9f08-3b5782a4d312.png">

But strange top 1% share result. I'm not sure if low N from filtering to WA can entirely explain this.
<img width="767" alt="image" src="https://user-images.githubusercontent.com/6076111/178296896-9349b529-6661-479e-88f1-f533b1eb1b3e.png">

## WA capital gains tax

The budgetary impact is surely distorted by the CPS's truncation, given it has a $250k standard deduction.
<img width="1218" alt="image" src="https://user-images.githubusercontent.com/6076111/178297604-d4e586b9-94a3-414d-84e4-ad13bf6d6ac4.png">

Increases inequality as expected:
<img width="873" alt="image" src="https://user-images.githubusercontent.com/6076111/178297933-f053b351-1625-4f32-b567-d09e8fb03cef.png">

## MA dependent care credit

I think this has a small impact by itself because filers can only either claim it or the dependent credit, so the dependent credit fills in the gap of its repeal.
<img width="1307" alt="image" src="https://user-images.githubusercontent.com/6076111/178293015-dd00c74b-957d-4315-815e-e18cb8917e52.png">

nikhilwoodruff commented:
OK, so the itemized page breaks because the SALT cap parameter doesn't have a breakdown specified.

nikhilwoodruff commented:
The standard deduction page is breaking because the standard deduction parameter isn't named by metadata.

MaxGhenis commented:
>The standard deduction page is breaking because the standard deduction parameter isn't named by metadata.

I'm still missing what's wrong. [Here's what we have](https://github.com/PolicyEngine/openfisca-us/blob/7516982bac4a541f02ba7e806b3d947974131c08/openfisca_us/parameters/gov/irs/deductions/standard/amount.yaml#L32):
```python
metadata:
  unit: currency-USD
  label: Standard deduction
  name: standard_deduction
  period: year
  breakdown:
    - filing_status
```

MaxGhenis commented:
* Bumped up the time estimate from 10 to 20 seconds based on a time test of the live site.
* Removed the deductions until we can figure out why they're breaking.
* Removed #822 as bumping OpenFisca US didn't fix the age chart locally
<img width="878" alt="image" src="https://user-images.githubusercontent.com/6076111/178824733-33a9607d-3bb8-412b-b0d9-fcd543936191.png">
